app/routes.js

```diff
@@ -136,12 +136,27 @@ router.get('/checkAnswersPdf', (req, res) => {
             res.setHeader('Content-disposition', 'attachment; filename=checkYourAnswers.pdf');
             res.send(result);
         })
+        .catch(err => {
+            res.status(500).render('errors/500', {cmmon: commonContent});
+        });
+    ;
+});
+
+router.get('/declarationPdf', (req, res) => {
+    const formdata = req.session.form;
+    services.createDeclarationPdf(formdata)
+        .then(result => {
+            res.setHeader('Content-Type', 'application/pdf');
+            res.setHeader('Content-disposition', 'attachment; filename=legalDeclaration.pdf');
```

Comment:
Would prefer this filename to use lowercase with hyphens, for example `legal-declaration.pdf`

Reply:
ok, done and now resides within documentDownload.js

Reply:
Can this please be changed for `checkYourAnswers.pdf` and `legalDeclaration.pdf`